### PR TITLE
Add vote for 2022/05/24: Merge pack round 1

### PR DESCRIPTION
**Pull request by folkyatina**
### Voting and tests for merge-ready protocol upgrade (round 1)

1. Publishing new implementation (0x47EbaB13B806773ec2A2d16873e2dF770D130b50) in Lido app APM repo
2. Updating implementation of the Lido app with the new one
3. Publishing new implementation (0x5d39ABaa161e622B99D45616afC8B837E9F19a25) in Node Operators Registry app APM repo
4. Updating implementation of Node Operators Registry app with the new one
5. Publishing new implementation (0x1430194905301504e8830ce4B0b0df7187E84AbD) in Oracle app APM repo
6. Updating implementation of Oracle app with a new one
7. Call Oracle's finalizeUpgrade_v3() to update the internal version counter.
8. Create permission for SET_EL_REWARDS_VAULT_ROLE of the Lido app assigning it to Voting
9. Create permission for STAKING_CONTROL_ROLE of the Lido app assigning it to Voting
10. Set execution layer rewards vault on Lido
    to new LidoExecutionLayerRewardsVault (0x388C818CA8B9251b393131C08a736A67ccB19297)
11. Resume staking on Lido
12. Set staking limit rate roughly to 150,000 ETH per day on Lido.

#### Context
- [Announcement on the Lido research forum](https://research.lido.fi/t/announcement-merge-ready-protocol-service-pack/2184)
- [Snapshot vote](https://snapshot.org/#/lido-snapshot.eth/proposal/0x37f68adc95bf8adb19dc14c0f0734bc4f6330274d3e63fbafae0a12373e8e997)
- [Twitter announcements](https://twitter.com/LidoFinance/status/1526121726639316994) :bird: 
- [Contracts](https://github.com/lidofinance/lido-dao/releases/tag/v1.2.0-rc.1)

### CODE REVIEW

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM except for some improvements.

File: scripts/vote_2022_05_24.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+update_oracle_app = {
+    'new_address': '0x1430194905301504e8830ce4B0b0df7187E84AbD',
+    'content_uri': '0x697066733a516d554d506669454b71354d786d387932475951504c756a47614a69577a31747665703557374564414767435238',
+    'id': '0x8b47ba2a8454ec799cd91646e7ec47168e91fd139b23f017455f3e5898aaba93',
+    'version': (3, 0, 0),
+}
+
+if network_name() in ("goerli", "goerli-fork"):
```
Comment (ujenjt) [inline scripts/vote_2022_05_24.py:66]: Maybe we can remove this?

File: tests/test_2022_05_24.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                                   validate_staking_limit_set)
+
+
+@pytest.fixture(scope="module")
+def deployer(accounts):
+    return accounts[2]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def deployed_contracts(deployer):
```
Comment (ujenjt) [inline tests/test_2022_05_24.py:26]: Let's remove this?

**Comment by ujenjt** [comment]
LGTM 👍

**Review by TheDZhon — COMMENTED** [review]
LGTM

**Review by TheDZhon — APPROVED** [review]

**Pull request merged**